geomesa-features/geomesa-feature-avro/src/main/scala/org/locationtech/geomesa/features/avro/AvroSimpleFeatureUtils.scala

```diff
@@ -29,6 +29,7 @@ object AvroSimpleFeatureUtils {
 
   val FEATURE_ID_AVRO_FIELD_NAME: String = "__fid__"
   val AVRO_SIMPLE_FEATURE_VERSION: String = "__version__"
+  val AVRO_SIMPLE_FEATURE_USERDATA: String = "__userdata__"
 
   // Increment whenever encoding changes and handle in reader and writer
   val VERSION: Int = 2
```

Comment:
doesn't matter to me... i assume a version 1 AvroDataFile can have version 2 avro simple features? seems like they're logically separate to me - one is the file version, the other is the feature version (if that's not the case, then yeah let's keep them in sync).